### DISCUSSION TITLE
How to automatically refresh the table after the save method?

### DISCUSSION THREAD

**Discussion by nerisonpitogo** [rappasoft/laravel-livewire-tables#1596] [Q&A]
I am very new to this package and I am having difficulty understanding the doc. How to automatically reset the table after the saving the record? 

    public function save_user()
    {
        $this->validate([
            'name' => 'required',
            'email' => 'required|email|unique:users,email,' . $this->user_id,
        ]);
        if ($this->user_id) {
            $user = User::find($this->user_id);
            $user->update([
                'name' => $this->name,
                'email' => $this->email,
            ]);
        } else {
            // generate a random password using password has in laravel with random string
            $random = Str::random(15);
            $password = Hash::make($random);

            User::create([
                'name' => $this->name,
                'email' => $this->email,
                'password' => $password,
            ]);
        }
        $this->dispatch('closeModal');
    }

I tried adding $this->emit('refreshDatatable'); 
but I get an error.

**Comment by lrljoe** [comment]
@nerisonpitogo  -> are you using V2 or V3?
